pkg/instances/instances.go

```diff
@@ -53,9 +53,12 @@ func (i *instances) Logs(id string) string {
 		return ""
 	}
 
+	const daprEnabledAnnotation string = "dapr.io/enabled"
+	const daprIDAnnotation string = "dapr.io/id"
```

Comment:
you can use it in one const declaration block ... and you can remove "string" type

const (
daprEnabledAnnotation string = "dapr.io/enabled"
daprIDAnnotation string = "dapr.io/id"
)